transformer.go

```diff
@@ -2,22 +2,48 @@ package main
 
 import (
 	"encoding/base64"
+	"encoding/xml"
 	"github.com/pborman/uuid"
 )
 
-func transformLocation(t term) location {
-	tmeIdentifier := buildTmeIdentifier(t.RawID)
+func transformLocation(tmeTerm term, taxonomyName string) location {
+	tmeIdentifier := buildTmeIdentifier(tmeTerm.RawID, taxonomyName)
 
 	return location{
 		UUID:          uuid.NewMD5(uuid.UUID{}, []byte(tmeIdentifier)).String(),
-		CanonicalName: t.CanonicalName,
+		CanonicalName: tmeTerm.CanonicalName,
 		TmeIdentifier: tmeIdentifier,
 		Type:          "Location",
 	}
 }
 
-func buildTmeIdentifier(rawId string) string {
+func buildTmeIdentifier(rawId string, tmeTermTaxonomyName string) string {
 	id := base64.StdEncoding.EncodeToString([]byte(rawId))
-	taxonomyName := base64.StdEncoding.EncodeToString([]byte(TaxonomyName))
+	taxonomyName := base64.StdEncoding.EncodeToString([]byte(tmeTermTaxonomyName))
 	return id + "-" + taxonomyName
 }
+
+type locationTransformer struct {
+}
+
+func (*locationTransformer) UnMarshallTaxonomy(contents []byte) ([]interface{}, error) {
+	taxonomy := taxonomy{}
+	err := xml.Unmarshal(contents, &taxonomy)
+	if err != nil {
+		return nil, err
+	}
+	var interfaces []interface{} = make([]interface{}, len(taxonomy.Terms))
```

Comment:
can't this be simplified:
`interfaces := make([]interface{}, len(taxonomy.Terms))` ?